src/main/java/seedu/address/model/person/Person.java

```diff
@@ -302,6 +302,40 @@ public Deadline getDeadline() {
     }
 
     /**
+     * Sets date of last accrued date.
+     * @param lastAccruedDate must not be null.
+     */
+    public void setLastAccruedDate(Date lastAccruedDate) {
+        requireNonNull(lastAccruedDate);
+        this.lastAccruedDate = lastAccruedDate;
+    }
+
+    @Override
+    public Date getLastAccruedDate() {
+        return lastAccruedDate;
+    }
+
+    //@@author jelneo
+    /**
+     * Sets total debt of a person to the given Debt.
+     * @param debt must not be null.
+     */
+    public void setTotalDebt(Debt debt) {
+        this.totalDebt.set(requireNonNull(debt));
```

Comment:
Maybe you should throw an exception when trying to set totalDebt < currentDebt

Reply:
good suggestion